include/slang/numeric/ConstantValue.h

```diff
@@ -144,6 +144,7 @@ class ConstantValue {
     ConstantValue convertToReal() const;
     ConstantValue convertToShortReal() const;
     ConstantValue convertToStr() const;
+    ConstantValue convertToByteArray(bitwidth_t size, bool isSigned = false) const;
```

Comment:
I'd prefer not to default the isSigned... it really should always be provided (or it should always be true, if we only allow this conversion to actual "byte" arrays, since byte is signed in SystemVerilog.

Reply:
I agree. I will remove the default value for isSigned.